elastic/kibana
# Fix intermitting series state

## Summary
Fixes https://github.com/elastic/kibana/issues/115739
Fixes an issue where Old data shows when deleting all series and starting over


### Testing

On master
1. Delete all the series
2. and you will see that without applying, chart will go blank
3. Add a series and the chart old state will come back


This branch
1. Delte all the series
2. You can see data remain if you don't hit apply
3. Click Apply changes and chart should go blanck
4. now you can start adding series and chart shouldn't restore previous state.
